plugins/kotlin-as-java/src/main/kotlin/KotlinAsJavaPlugin.kt

```diff
@@ -18,6 +18,6 @@ class KotlinAsJavaPlugin : DokkaPlugin() {
                     dokkaBasePlugin.commentsToContentConverter
                 ), ctx.logger
             )
-        }
+        } applyIf { format != "javadoc" }
```

Comment:
I'd suggest `format == "kotlin-as-java"`

Reply:
Paweł's changes were merged so i've used the override mechanism